src/core_plugins/kibana/public/selectors/dashboard_selectors.js

```diff
@@ -1,19 +1,25 @@
 import * as DashboardSelectors from '../dashboard/selectors';
 
+/**
```

Comment:
I must have missed this before, but why does this file exist again? Is there anything outside of `public/dashboard` calling any of these selectors right now? Is the idea that they might in the future?

Reply:
It's because "core kibana" apps are bundled as a single app, even though they exist as separate links in the left nav.  This is to avoid the optimization screen when jumping between them.  So technically, as one big app with visualize, discover, and dashboard (and a few others I think...), there would be a single redux tree.  So I set it up at the root, even though dashboard is the only one using the store right now.

I figured this way was more future proof, so as soon as another core app wants to use redux, there doesn't have to be a refactoring first.

Reply:
Cool!